### PR TITLE
Lecture/adan api restful

### PR DESCRIPTION
**Pull request by Adanzar**

### CODE REVIEW

File: challenges/Adan/hero-rest-api/indexCharacter.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
```
Comment (gilbecao) [inline challenges/Adan/hero-rest-api/indexCharacter.js:1]: Merge with index.js

File: challenges/Adan/restful-adan-demo/controllers/heroesController.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = [
```
Comment (gilbecao) [inline challenges/Adan/restful-adan-demo/controllers/heroesController.js:1]: This is a constant

File: challenges/Adan/hero-rest-api/controller/ramController.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    );
+    maxHeroId = heroesOrdered[heroesOrdered.length - 1].id;
+  }());
+  return {
+    getAll: (req, res) => {
+      res.json(ramCharacters);
+    },
+    create: (req, res) => {
+      maxHeroId += 1;
+      const newHero = {
```
[the remark this answers is not in the document]
Comment (Adanzar) [reply challenges/Adan/hero-rest-api/controller/ramController.js:17]: done

File: challenges/Adan/restful-adan-demo/controllers/heroesController.js (hunk 2 above)
Comment (Adanzar) [reply challenges/Adan/restful-adan-demo/controllers/heroesController.js:1]: done

File: challenges/Adan/hero-rest-api/indexCharacter.js (hunk 1 above)
Comment (Adanzar) [reply challenges/Adan/hero-rest-api/indexCharacter.js:1]: done

File: challenges/Adan/restful-adan-demo/controllers/heroesController.js (hunk 2 above)
Comment (gilbecao) [reply challenges/Adan/restful-adan-demo/controllers/heroesController.js:1]: Esto no está done

**Review by gilbecao — APPROVED** [review]

**Pull request merged**